FormFrameworkREADME.md

```diff
@@ -0,0 +1,289 @@
+# Form Framework
+One of the benefits of the Form Framework tool is that it allows simple coupling of the front- and back-end sides when creating a new form.
+
+----
+## Table of Contents
+
+* [Step-by-Step Guide to Create a New Form](#step-by-step-guide-to-create-a-new-form)
+* [Modifying an Existing Schema](#modifying-an-existing-schema)
+* [Front-End Architecture Overview](#front-end-architecture-overview)
+* [Front-End keywords glossary](#front-end-keywords-glossary)
+* [Back-End keywords glossary](#back-end-keywords-glossary)
+----
+
+## Step-by-Step Guide to Create a New Form
+
+### 1. Create the .yaml file for the schema
+The starting point for a new form is the schema's .yaml file, which specifies the configuration of both front- and back-end components of the form.
+
+  1. If you haven't already done so, install [ts-node](https://www.npmjs.com/package/ts-node) by running `npm install ts-node -g`
+
+  1. Generate a skeleton .yaml file for the schema by running `ts-node api/api_sources/scripts/create.schema.ts -s <Schema-Name>` from the command line. The skeleton .yaml file will be placed within the `api/api_sources/schema-files` directory, with the name of the file matching the specified \<Schema-Name>.  
+  Alternatively, rather than creating a new schema file, you can create a new schema by adding it to an existing schema file (multiple schemas can be defined with a .yaml file). This is recommended if 2 or more schemas are related in some logical way.
+
+  1. Begin by populating the preliminary information for the schema:
+    - `name:` the name of the table as it will be stored in the DB. Should be in snake-case, with only lowercase letters
```

Comment:
I think the indentation is little off here, because in the preview I see these in a paragraph rather than as points. I'm assuming, this is not intentional

![image](https://user-images.githubusercontent.com/57013405/74961886-b3e3cb80-53c3-11ea-9839-99c97c6072b3.png)

Reply:
Ahhh yes. I was using Markdown Preview in Atom, but apparently Atom doesn't interpret markdown formatting in quite the same way that GitHub does. I think I've fixed it now. 
Thanks for catching that!